lib/java_buildpack/framework/dynatrace_one_agent.rb

```diff
@@ -191,7 +200,11 @@ def logstream?
       end
 
       def skip_errors?
-        credentials[SKIP_ERRORS].to_b
+        credentials[SKIP_ERRORS] == "true"
```

Comment:
```suggestion
        credentials[SKIP_ERRORS] == 'true'
```
To satisfy linter complaint